patterns/event-rsvp.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Title: Event RSVP
+ * Slug: twentytwentyfive/event-rsvp
+ * Keywords: call-to-action, rsvp, event
+ * Categories: call-to-action
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull">
+	<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"grid","minimumColumnWidth":"70rem"}} -->
+	<div class="wp-block-group alignfull">
+		<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"bottom":"var:preset|spacing|80","left":"var:preset|spacing|40","right":"var:preset|spacing|40","top":"var:preset|spacing|40"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"top"}} -->
+		<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--40);padding-right:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--80);padding-left:var(--wp--preset--spacing--40)">
+			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+			<div class="wp-block-group">
+				<!-- wp:heading {"level":1} -->
+				<h1 class="wp-block-heading">Stories, historias, iсторії, iστορίες</h1>
```

Comment:
Is this a landing page meant for a "page with no title"? Otherwise we need to avoid using the H1's inside the patterns. Because there will be two H1's and the pre-made designs should help the users avoid that.

Reply:
It would be most suitable for a page with no title, yes. If that's the case, we can make it H2 but using the same (or the most similar) font size.